### PR TITLE
[Components with more than 1 service] LPS-195845 Refactor ClusterExecutorImpl component

### PR DESCRIPTION
**Pull request by RenanUmV**

### CODE REVIEW

**Comment by RenanUmV** [comment]
ci:test:sf

**Comment by RenanUmV** [comment]
ci:test:relevant

**Comment by RenanUmV** [comment]
ci:test:clustering

File: modules/apps/portal/portal-cluster-multiple/src/main/java/com/liferay/portal/cluster/multiple/internal/ClusterExecutorImpl.java [hunk 1]
```diff
@@ -671,4 +673,92 @@ public void portalServerInetSocketAddressConfigured(
 
 	}
 
+	private class ClusterRequestReceiver extends BaseClusterReceiver {
```
Comment (rafaprax) [inline modules/apps/portal/portal-cluster-multiple/src/main/java/com/liferay/portal/cluster/multiple/internal/ClusterExecutorImpl.java:676]: Hey @RenanUmV, you do not need to inline this class. Also, there is still a @Reference for the `ClusterExecutorImpl` here modules/apps/portal/portal-cluster-multiple/src/main/java/com/liferay/portal/cluster/multiple/internal/ClusterMasterExecutorImpl.java

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by rafaprax** [comment]
@RenanUmV see the above comment and resend.

**Pull request closed without merging**